src/bun_cpp_utils.cpp

```diff
@@ -5,23 +5,30 @@
 
 #include <cxxabi.h>
 
+template<typename T>
+struct deleter
```

Comment:
the struct's name should reflect what it does (it calls `free(2)`), not only how it's used.